pages/tutorials/yaml/phases.rst

```diff
@@ -0,0 +1,382 @@
+species involved are declared as being in the phase, the ``reactions`` entry
+can be set to the string ``declared-species``.
+
+To include reactions from multiple sections or other files, the ``reactions``
+entry can be given as a list of section names, for example:
+
+.. code:: yaml
+
+    reactions:
+    - OH_submechanism
+    - otherfile.yaml/C1-reactions
+    - otherfile.yaml/C2-reactions
+
+To include reactions from multiple sections or other files while only including
+reactions involving declared species, a list of single-key mappings can be used,
+where the key is the section name (or file and section name) and the value is
+either the string ``all`` or the string ``declared-species``. For example:
+
+.. code:: yaml
+
+    reactions:
+    - OH_submechanism: all
+    - otherfile.yaml/C1-reactions: all
+    - otherfile.yaml/C2-reactions: declared-species
+
+To permit reactions containing third-body efficiencies for species not present
+in the phase, the additional field ``skip-undeclared-third-bodies`` may be added
+to the phase entry with the value ``true``.
+
+Setting the Transport Model
+---------------------------
+
+To enable transport property calculation, the transport model to be used can be
+specified in the ``transport`` field. Supported models are:
+
+    - `high-pressure <{{% ct_docs doxygen/html/d9/d63/classCantera_1_1HighPressureGasTransport.html#details %}}>`__
+    - `ionized-gas <{{% ct_docs doxygen/html/d4/d65/classCantera_1_1IonGasTransport.html#details %}}>`__
+    - `mixture-averaged <{{% ct_docs doxygen/html/d9/d17/classCantera_1_1MixTransport.html#details %}}>`__
+    - `mixture-averaged-CK <{{% ct_docs doxygen/html/d9/d17/classCantera_1_1MixTransport.html#details %}}>`__
+    - `multicomponent <{{% ct_docs doxygen/html/df/d7c/classCantera_1_1MultiTransport.html#details %}}>`__
+    - `multicomponent-CK <{{% ct_docs doxygen/html/df/d7c/classCantera_1_1MultiTransport.html#details %}}>`__
+    - `unity-Lewis-number <{{% ct_docs doxygen/html/d3/dd6/classCantera_1_1UnityLewisTransport.html#details %}}>`__
+    - `water <{{% ct_docs doxygen/html/df/d1f/classCantera_1_1WaterTransport.html#details %}}>`__
+
+
+Setting the Initial State
+-------------------------
+
+The state of a can be set using two properties to set the thermodynamic state,
+plus the composition. This state is specified as a mapping in the ``state``
+field of ``phase`` entry.
+
+The composition can be set using one of the following fields, depending on the
+phase type. The composition is specified as a mapping of species names to
+values. Where necessary, the values will be automatically normalized.
+
+- ``mass-fractions`` or ``Y``
+- ``mole-fractions`` or ``X``
+- ``coverages``
+- ``molalities`` or ``M``
+
+The thermodynamic state can be set in terms of two of the following properties,
+with the valid property pairs deplending on the phase model:
+
+- ``temperature`` or ``T``
+- ``pressure`` or ``P``
+- ``enthalpy`` or ``H``
```

Comment:
I'll update the discussion of `Q` pending the resolution of https://github.com/Cantera/cantera/pull/784.

The naming logic was to use the uppercase names for consistency with the state setters like `HPX`, which use capital letters, rather than the individual properties like `h`. We could make those particular names case insensitive.